cameo/api/designer.py

```diff
@@ -73,7 +73,7 @@ def __call__(self, strategy):
 
 class _OptGeneRunner(_OptimizationRunner):
     def __call__(self, strategy):
-        max_evaluations = 20000
+        max_evaluations = 15000
```

Comment:
you don't want to make this an argument instead? Changing like this seems cumbersome?